### PR TITLE
Show fixed speed in mania scroll speed settings

### PR DESCRIPTION
**Pull request by cdwcgt**
a small change

related: #20102 

image

<details><summary>old ver</summary>

X)CP3E2 @L~NO521RWMMJUL

</details>

### CODE REVIEW

File: osu.Game.Rulesets.Mania/ManiaSettingsSubsection.cs [hunk 1]
```diff
@@ -47,5 +46,10 @@ private void load()
                 }
             };
         }
+
+        private class ManiaScorllSlider : OsuSliderBar<double>
```
Comment (jai-x) [inline osu.Game.Rulesets.Mania/ManiaSettingsSubsection.cs:50]: Typo `ManiaScorllSlider` -> `ManiaScrollSlider`

Comment (cdwcgt) [reply osu.Game.Rulesets.Mania/ManiaSettingsSubsection.cs:50]: ouch
I'll fix it right away

**Comment by peppy** [comment]
Hmm, I'm not sure about the display method here. @smoogipoo thoughts? I mentioned that it should display the same as the HUD overlay thing, but it makes less sense when the integer values don't match with the slider bar's adjustment itself. Maybe back-to-front makes more sense (ie `500ms (speed 1)`)?

**Comment by cdwcgt** [comment]
Modify the text of the HUDoverlay to `ms (fixspeed)` too?
because lazer no longer uses fixed speed rate as the main unit
Displaying fixed speeds is just for stable transitions

**Comment by smoogipoo** [comment]
Originally I was planning to say that I'm not sure speed needs to/should be incrementally adjusted as opposed to using the integer ticks. But if it must be then probably `500ms (speed 1)` is better.

**Comment by smoogipoo** [comment]
That's fine by me

**Review by peppy — APPROVED** [review]

**Pull request merged**